language-server/src/main/java/org/sonarlint/languageserver/RuleDescription.java

```diff
@@ -90,17 +104,27 @@ public RuleDescription setSeverity(String severity) {
     return this;
   }
 
+  public RuleDescription setActiveByDefault(boolean activeByDefault) {
+    this.activeByDefault = activeByDefault;
+    return this;
+  }
+
   @Override
   public String toString() {
     return MessageJsonHandler.toString(this);
   }
 
-  public static RuleDescription of(String key, String name, String htmlDescription, String type, String severity) {
+  public static RuleDescription of(String key, String name, @Nullable String htmlDescription, @Nullable String type, String severity, boolean activeByDefault) {
```

Comment:
Do we really have rules with null description/type?

Reply:
I don't think so, but they have `@CheckForNull` on `org.sonarsource.sonarlint.core.client.api.common.RuleDetails`

Reply:
OK, fine then